docs/cookbook/deploy-with-kamal.md

```diff
@@ -0,0 +1,222 @@
+# Deploy with `Kamal`
+
+Rails 8 will ship with [Kamal](https://kamal-deploy.org/) preconfigured as the default deployment tool.
```

Comment:
This is true when you generate a new app with `rails new -j esbuild` or other builders. By default (with importmap), you'll be missing Node.js-related dependencies.

Reply:
Is this comment related to this line though ?
'Cause I'm pretty sure that Kamal is the default deployment tool for rails 8, even without `-j esbuild` flag

Reply:
@skryukov I did update my PR to highlight the difference of example Dockerfile compared to the fresh default Rails 8. The preview of the new docs have been updated in the PR description as well. Could you help me to re-review it, please?